mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
+	METHOD readSlice readSlice (I)Lio/netty/buffer/ByteBuf;
+		ARG 1 length
+	METHOD readUnsignedByte readUnsignedByte ()S
+	METHOD readUnsignedInt readUnsignedInt ()J
+	METHOD readUnsignedIntLE readUnsignedIntLE ()J
+	METHOD readUnsignedMedium readUnsignedMedium ()I
+	METHOD readUnsignedMediumLE readUnsignedMediumLE ()I
+	METHOD readUnsignedShort readUnsignedShort ()I
+	METHOD readUnsignedShortLE readUnsignedShortLE ()I
+	METHOD readableBytes readableBytes ()I
+	METHOD readerIndex readerIndex ()I
+	METHOD readerIndex readerIndex (I)Lio/netty/buffer/ByteBuf;
+		ARG 1 i
+	METHOD refCnt refCnt ()I
+	METHOD release release ()Z
+	METHOD release release (I)Z
+		ARG 1 decrement
+	METHOD resetReaderIndex resetReaderIndex ()Lio/netty/buffer/ByteBuf;
+	METHOD resetWriterIndex resetWriterIndex ()Lio/netty/buffer/ByteBuf;
+	METHOD retain retain ()Lio/netty/util/ReferenceCounted;
+	METHOD retain retain (I)Lio/netty/util/ReferenceCounted;
+		ARG 1 increment
+	METHOD retainedDuplicate retainedDuplicate ()Lio/netty/buffer/ByteBuf;
+	METHOD retainedSlice retainedSlice ()Lio/netty/buffer/ByteBuf;
+	METHOD retainedSlice retainedSlice (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 length
+	METHOD setBoolean setBoolean (IZ)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setByte setByte (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setBytes setBytes (ILio/netty/buffer/ByteBuf;)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 buffer
+	METHOD setBytes setBytes (ILio/netty/buffer/ByteBuf;I)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 buffer
+		ARG 3 length
+	METHOD setBytes setBytes (ILio/netty/buffer/ByteBuf;II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 buffer
+		ARG 3 sourceIndex
+		ARG 4 length
+	METHOD setBytes setBytes (ILjava/io/InputStream;I)I
+		ARG 1 index
+		ARG 2 inputStream
+		ARG 3 length
+	METHOD setBytes setBytes (ILjava/nio/ByteBuffer;)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 buffer
+	METHOD setBytes setBytes (ILjava/nio/channels/FileChannel;JI)I
+		ARG 1 index
+		ARG 2 fileChannel
+		ARG 3 position
+		ARG 5 length
+	METHOD setBytes setBytes (ILjava/nio/channels/ScatteringByteChannel;I)I
+		ARG 1 index
+		ARG 2 scatteringByteChannel
+		ARG 3 length
+	METHOD setBytes setBytes (I[B)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 bytes
+	METHOD setBytes setBytes (I[BII)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 bytes
+		ARG 3 sourceIndex
+		ARG 4 length
+	METHOD setChar setChar (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setCharSequence setCharSequence (ILjava/lang/CharSequence;Ljava/nio/charset/Charset;)I
+		ARG 1 index
+		ARG 2 charSequence
+		ARG 3 charset
+	METHOD setDouble setDouble (ID)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setFloat setFloat (IF)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setIndex setIndex (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 oldIndex
+		ARG 2 newIndex
```

Comment:
From [Conventions.md: Getters, setters, withers, and creators](https://github.com/QuiltMC/quilt-mappings/blob/1.18.2/CONVENTIONS.md#getters-setters-withers-and-creators)
> Name the parameter the same as the property (`setColor(color)`, not `setColor(newColor)`).

```suggestion
		ARG 2 index
```